server/src/test/java/org/opensearch/discovery/InitializeExtensionRequestTests.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.discovery;
+
+import org.opensearch.Version;
+import org.opensearch.cluster.node.DiscoveryNode;
+import org.opensearch.cluster.node.DiscoveryNodeRole;
+import org.opensearch.common.bytes.BytesReference;
+import org.opensearch.common.io.stream.BytesStreamInput;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.common.transport.TransportAddress;
+import org.opensearch.extensions.DiscoveryExtensionNode;
+import org.opensearch.extensions.ExtensionDependency;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.net.InetAddress;
+import java.util.HashMap;
+import java.util.List;
+
+public class InitializeExtensionRequestTests extends OpenSearchTestCase {
+
+    public void testInitializeExtensionRequest() throws Exception {
+        String expectedUniqueId = "test uniqueid";
+        Version expectedVersion = Version.fromString("2.0.0");
+        ExtensionDependency expectedDependency = new ExtensionDependency(expectedUniqueId, expectedVersion);
+        DiscoveryExtensionNode expectedExtensionNode = new DiscoveryExtensionNode(
+            "firstExtension",
+            "uniqueid1",
+            new TransportAddress(InetAddress.getByName("127.0.0.0"), 9300),
+            new HashMap<>(),
+            Version.fromString("3.0.0"),
+            Version.fromString("3.0.0"),
```

Comment:
Backporting this test to 2.x might fail because the minimum compatibility version check is done by `DiscoveryExtensionNode`. [1] 
To have the same piece of code working on both branches you could use `Version.CURRENT`.[2] 

[1] https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/extensions/DiscoveryExtensionNode.java#L106
[2] https://github.com/opensearch-project/OpenSearch/blob/2.x/server/src/test/java/org/opensearch/extensions/ExtensionsManagerTests.java#L289

Reply:
changed to Version.CURRENT

Thanks